src/loop.cpp

```diff
@@ -6,11 +6,12 @@
 #include "lfo.h"
 #include "sid.h"
 #include "ui.h"
+#include <EEPROM.h>
 
 static bool loadedAfterStartup; // we load the preset after 2sec (when SID is ready)
 
 static int calc_pitch(int note, float frac) {
-	static const int32_t sidScale[] = {
+	/*static const int32_t sidScale[] = {
```

Comment:
When using git, there is no need to comment out unused code. Just delete it from the file.

You can always go back in git's commit history.

Reply:
It appears that you haven't yet deleted this comment. I highly recommend to delete code instead of commenting it out.

Reply:
did you forget to push (or to commit)? ;)